client/my-sites/checkout/composite-checkout/test/wp-checkout-order-summary.tsx

```diff
@@ -32,13 +34,9 @@ jest.mock( '@automattic/calypso-config', () => {
 	return mock;
 } );
 
-const jetpackBackupFeatureList = [
-	'Real-time cloud backups',
-	'10GB of backup storage',
-	'30-day archive & activity log',
-	'One-click restores',
-	'Priority support',
-];
+const translate: ( original: string ) => string = ( original ) => {
+	return original;
+};
```

Comment:
In case you didn't know, a function like this is often called an "identity" function. While I see you're trying to base it off the existing function type, I think you could simplify the stub even more to make it more readable.

See this other test that does the same stub of the translate function as an example: https://github.com/Automattic/wp-calypso/blob/b55a5036225d4faf6b94e0eff474816fa8e3b5e1/client/my-sites/plan-features/test/index.jsx#L43

Reply:
Thanks for the review and the tip! Got that updated 😄